docs/b2b-edition/specs/api-v3/sales_staff/sales_staff.yaml

```diff
@@ -1,212 +1,176 @@
-openapi: 3.0.0
+openapi: 3.0.1
 info:
   title: Sales Staff
-  version: '3.0'
-  description: Sales staff management
-  contact: {}
+  description: |-
```

Comment:
Is it worth a note explicitly calling out that Sales Staff are different than Super Admins?  I feel like it's a common mistake to conflate the two and/or expect Super Admins to be admin users.